services/director-v2/src/simcore_service_director_v2/modules/db/repositories/sharing_networks.py

```diff
@@ -0,0 +1,23 @@
+import sqlalchemy as sa
+from aiopg.sa.result import RowProxy
+from models_library.projects import ProjectID
+from models_library.sharing_networks import SharingNetworks
+
+from ....core.errors import ProjectNotFoundError
+from ..tables import sharing_networks
+from ._base import BaseRepository
+
+
+class SharingNetworksRepository(BaseRepository):
+    async def get_sharing_networks(self, project_id: ProjectID) -> SharingNetworks:
```

Comment:
I see you get the sharing networks. but where do you write to the table??

Reply:
Inside the webserver they are written to the table. The update is done each time a change to the project occurs. Once that happens the project networks are calculated (currently only the default). All dynamic-sidecar nodes are placed on this network.